src/components/Admin/ManagementPageTemplate.tsx

```diff
@@ -17,16 +17,33 @@ interface Props extends PageProps {
       url: string;
     }>;
   }>;
+  title?: string;
+  returnUrl?: string;
 }
 
-export const ManagementPageTemplate: FC<Props> = ({ data, paths }) => {
+export const ManagementPageTemplate: FC<Props> = ({ title, returnUrl, data, paths }) => {
   const { url } = useRouteMatch();
 
   useUpdateNavigation({ currentPaths: paths });
 
+  const titleElement = useMemo(
+    () =>
+      returnUrl ? (
+        <Link component={RouterLink} to={returnUrl}>
+          {title}
+        </Link>
+      ) : (
+        title
```

Comment:
What is the point of having just the text?

Reply:
If the link is missing the "Title" still can be shown.

Reply:
That was exactly my point. Why do we need to show the title there?

Reply:
It makes the settings page more complete